packages/mobile/src/localCurrency/selectors.ts

```diff
@@ -50,16 +51,30 @@ export function getLocalCurrencySymbol(state: RootState): LocalCurrencySymbol |
   return LocalCurrencySymbol[getLocalCurrencyCode(state)]
 }
 
-export function getLocalCurrencyExchangeRate(state: RootState) {
-  const { exchangeRate, fetchedCurrencyCode } = state.localCurrency
+export function localCurrencyExchangeRateSelector(state: RootState) {
+  const {
+    exchangeRate,
+    eurExchangeRate,
+    celoExchangeRate,
+    fetchedCurrencyCode,
+  } = state.localCurrency
 
   const localCurrencyCode = getLocalCurrencyCode(state)
   if (localCurrencyCode !== fetchedCurrencyCode) {
     // This makes sure we don't return stale exchange rate when the currency code changed
-    return null
+    return {}
   }
 
-  return exchangeRate
+  return {
+    [Currency.Celo]: celoExchangeRate,
+    [Currency.Dollar]: exchangeRate,
+    [Currency.Euro]: eurExchangeRate,
+  }
+}
+
+export function getLocalCurrencyExchangeRate(state: RootState) {
```

Comment:
Yes, I initially held off on this because I planned to deprecate this function since most of the uses will go away with all the cEUR work, but I guess it makes sense to do it now.